### PR TITLE
Add production sample with prebuilt workflow bundle

### PR DESCRIPTION
**Pull request by lorensr**
Wasn't sure where to put the bundle. Chose `/workflow-bundle.js`. Didn't like it in `lib/` because it got overwritten on `npm build`.

## Why?

So we have a full working sample we can point to from the new deployment docs: https://github.com/temporalio/documentation/pull/719

### CODE REVIEW

File: production/scripts/build-workflow-bundle.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import path from 'path';
+
+async function bundle() {
+  const { code } = await bundleWorkflowCode({
+    workflowsPath: require.resolve('../src/workflows'),
+  });
+
+  await writeFile(path.join(__dirname, '../workflow-bundle.js'), code);
+
+  console.log('Bundle written to workflow-bundle.js');
```
Comment (bergundy) [inline production/scripts/build-workflow-bundle.ts:12]: nit: might as well print the entire path here.

File: production/src/worker.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Worker } from '@temporalio/worker';
+import * as activities from './activities';
+
+async function run() {
+  const worker = await Worker.create({
+    ...(process.env.NODE_ENV === 'production'
```
Comment (bergundy) [inline production/src/worker.ts:6]: nit: Maybe extract this to a function?

**Review by bergundy — APPROVED** [review]
Just the snippets need to be removed, nothing else that's blocking

File: production/src/workflows.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// @@@SNIPSTART typescript-hello-workflow
```
[the remark this answers is not in the document]
Comment (lorensr) [reply production/src/workflows.ts:1]: 😄

**Pull request merged**